README.md

```diff
@@ -156,6 +156,28 @@ restore this behaviour with the `--workdir` option:
 docker run [...] --workdir=/home/mermaidcli minlag/mermaid-cli [...]
 ```
 
+## Use Podman:
+
+Pull from Github Container Registry
+
+```sh
+podman pull ghcr.io/mermaid-js/mermaid-cli/mermaid-cli
+```
+
+The container looks for input files in `/data`. So for example, if you have a
+diagram defined on your system in `/path/to/diagrams/diagram.mmd`, you can use
+the container to generate an SVG file as follows:
+
+```sh
+podman run --userns keep-id --user ${UID} --rm -v /path/to/diagrams:/data:Z ghcr.io/mermaid-js/mermaid-cli/mermaid-cli -i diagram.mmd
```

Comment:
Thanks for making this PR! I prefer using Podman instead of Docker, so I'd probably use these instructions :smile:

Can you explain why this command is different from the Docker command in https://github.com/mermaid-js/mermaid-cli/blob/7bcf75065f7b49386f6784826e9c8410e99115a5/README.md?plain=1#L148-L150

If that's the only difference between `## Using Docker`/`## Using Podman`, do you think it makes sense to combine the two sections?

E.g. we could do something like:

````markdown
## Use Docker/Podman

<!-- Snip -->

```sh
docker pull minlag/mermaid-cli:8.8.0
```

The container looks for input files in `/data`. So for example, if you have a
diagram defined on your system in `/path/to/diagrams/diagram.mmd`, you can use
the container to generate an SVG file as follows:

```sh
docker run --rm -u `id -u`:`id -g` -v /path/to/diagrams:/data minlag/mermaid-cli -i diagram.mmd
```

Or, if using [Podman](https://podman.io/), instead do:

```sh
podman run --userns keep-id --user ${UID} --rm -v /path/to/diagrams:/data:Z ghcr.io/mermaid-js/mermaid-cli/mermaid-cli -i diagram.mmd
```
````

Reply:
Combined the sections and added explanation of the difference to the readme.